zuul/tenants.yaml

```diff
@@ -74,6 +74,7 @@
               - ansible/awx
               - ansible/galaxy
               - ansible/molecule
+              - ansible/workshops
```

Comment:
we should also include the change to github here too, so we can ensure config on github side is setup properly